mosaic-runtime/src/commonTest/kotlin/com/jakewharton/mosaic/AnsiRenderingTest.kt

```diff
@@ -178,11 +178,11 @@ class AnsiRenderingTest {
 
 		assertThat(rendering.render(rootNode).toString()).isEqualTo(
 			"""
-			|One
-			|Two
+			|One $s
+			|Two $s
```

Comment:
Yes, I know it looks worse here. I did this consciously, because I made https://github.com/JakeWharton/mosaic/pull/439 where I delete empty characters at the end of lines, which, in my opinion, eliminates this problem later.

Reply:
I think trimming at the end makes sense for an individual surface, especially the main one. I'll get to that PR soon.

But by combining the static surfaces we're now doing more work to merge them and allocate more space, only to then have to process more to trim the new trailing spaces. And ultimately it results in no change in the output but more code to maintain.

Since in practice there aren't multiple static nodes, and even if there are they're all not rendering at the same time, and even if multiple render they're not rendering much, doing this still doesn't gain us anything.